src/main/java/com/epam/emotionalhelp/controller/response/ResponseHandler.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.emotionalhelp.controller.response;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ResponseHandler {
+    public static ResponseEntity<Object> generateResponse(String message, HttpStatus status, Object responseObj) {
+        Map<String, Object> map = new HashMap<>();
+        map.put("message", message);
+        map.put("status", status.value());
+        map.put("data", responseObj);
+
+        return new ResponseEntity<>(map, status);
+    }
+
+    public static ResponseEntity<Object> generateResponse(String message, HttpStatus status) {
+        Map<String, Object> map = new HashMap<>();
+        map.put("message", message);
+        map.put("status", status.value());
```

Comment:
Should we introduce a new entity for response?

Reply:
Maxim can you please explain in a few words?